src/Common/src/Interop/Unix/libssl/Interop.SafeSslHandle.cs

```diff
@@ -136,7 +140,11 @@ protected override bool ReleaseHandle()
 
             private SafeSslHandle() : base(IntPtr.Zero, true)
             {
-            }   
+            }
+
+            public SafeSslHandle(IntPtr sslPointer) : base(sslPointer, false)
```

Comment:
The (`IntPtr`) constructor usually implies `ownsHandle: true`.  Please make `ownsHandle` still be a parameter here and specify `false` at the callsite.